msu/utilities/tile.nut

```diff
@@ -0,0 +1,15 @@
+::MSU.Tile <- {
+	function canResurrectOnTile( _tile, _force = false )
```

Comment:
We should discuss if we want to keep this function in MSU. Other than this, can you think of any other tile utilties? Maybe some functions to get adjacent tiles based on certain conditions? I'd especially like to condense the vanilla method of
```
if (tile.hasNextTile(i)) local nextTile = tile.getNextTile(i);
```
to something like
```
local nextTile = ::MSU.Tile.getNextTile(_tile, i);
```
which returns null if there is no next tile, otherwise returns the tile.

Perhaps we can think of som eother functions too, for example based on tile direction etc.

Reply:
I think this could easily run into issues where nextTile is unexpectedly null.

Reply:
No. Because ::MSU.Tile.getNextTile(_tile, i) will `return _tile.hasNextTile(i) ? _tile.getNextTile(i) : null;`

Reply:
Yes, but then you still need to check `if (nextTile != null)`, so in effect you don't really gain anything, do you?

Reply:
Yeah you are correct. Ok then let's bring the discussion back to whether we want the `canResurrectOnTile` function in MSU.

Reply:
Removed for now